src/Components/Users/UserAdd.tsx

```diff
@@ -597,14 +594,18 @@ export const UserAdd = (props: UserProps) => {
     };
   };
 
+  if (isLoading) {
+    return <Loading />;
+  }
+
   return (
     <Page
       title={headerText}
       options={
         <Link
-          href="https://school.coronasafe.network/targets/12953"
+          href="https://school.coronasafe.network/targets/12953 "
```

Comment:
Why was this changed?

```suggestion
          href="https://school.coronasafe.network/targets/12953"
```

Reply:
While saving the file extra space removed.
Is this making any conflict?